File: src/nonim/backend/test/cases/phase0/expected.untyped.c
static int const max = 10;
static int count = 0;
static int const step = 1;
static int increment (int const value, int const amount) {
  return value + amount;
}
static int accumulate (int const limit) {
  int total = 0;
  int current = 0;
  while (current < limit) {
    total = increment(total, step);
    current = current + 1;
    if (current == 5) {
      (void)(increment(current, 0));
      continue;
    }
    if (total > 100) {
      break;
    }
  }
  return total;
}
static int run () {
  int const output = accumulate(max);
  (void)(count);
  return output;
}

File: src/nonim/backend/test/cases/phase0/expected.untyped.zig
pub const max: isize = 10;
pub var count: isize = 0;
pub const step: isize = 1;
pub fn increment (value: isize, amount: isize) isize {
  return value + amount;
}
pub fn accumulate (limit: isize) isize {
  var total: isize = 0;
  var current: isize = 0;
  while (current < limit) {
    total = increment(total, step);
    current = current + 1;
    if (current == 5) {
      _ = increment(current, 0);
      continue;
    }
    if (total > 100) {
      break;
    }
  }
  return total;
}
pub fn run () isize {
  const output: isize = accumulate(max);
  _ = count;
  return output;
}
